internal/cli/inputs_test.go

```diff
@@ -218,14 +226,24 @@ func TestResolveGroupID(t *testing.T) {
 			},
 			expectedGroupID: testGroup.ID,
 		},
+		{
+			description:     "Should select default group when FlagAutoConfirm is set",
+			groups:          []atlas.Group{testGroup, testGroup2},
+			procedure:       func(c *expect.Console) {},
+			autoConfirm:     true,
+			defaultGroupID:  testGroup2.ID,
+			expectedGroupID: testGroup2.ID,
+		},
 	} {
 		t.Run(tc.description, func(t *testing.T) {
 			atlasClient := mock.AtlasClient{}
 			atlasClient.GroupsFn = func() ([]atlas.Group, error) {
 				return tc.groups, nil
 			}
 
-			_, console, _, ui, consoleErr := mock.NewVT10XConsole()
+			console, _, ui, consoleErr := mock.NewVT10XConsoleWithOptions(
+				mock.UIOptions{AutoConfirm: tc.autoConfirm},
+				new(bytes.Buffer))
```

Comment:
[style] to keep things balanced, you can end this line with `,` and have the last trailing `)` on its own newline